### DISCUSSION TITLE
Change Sympa logo to our own

### DISCUSSION THREAD

**Discussion by UAProfessorProton** [sympa-community/sympa#1867] [Q&A] [answered]
We are using Sympa as list server in our organization, we wish to further customize it by changing the sympa logo with our own.  Is there an online documentation available to do this? I'm sorry for the newbie question. I can't find any applicable information from this page: https://www.sympa.community/manual/customize.html. Any recommended resolution size for the logo? 

Thank you in advance.

**Comment by ldidry** [comment, marked as answer]
You can change the logo with the [`logo_html_definition`](https://www.sympa.community/gpldoc/man/sympa_config.5.html#logo_html_definition) setting.

Like this:
```
logo_html_definition <a href="https://lists.exemple.org/sympa"><img src="/static-sympa/icons/my-custom-logo.png" alt="my custom logo"></a>
```
